### PR TITLE
Added option to add Https proxy agent

### PR DESCRIPTION
**Pull request by gelim-ags**
Will add agent (HttpsProxyAgent) to http request options if http_proxy env variable is present

### CODE REVIEW

**Comment by gelim-ags** [comment]
Hello, requesting followup for this PR :)

File: lib/api.js [hunk 1]
```diff
@@ -63,14 +65,17 @@ function _prepareOptionsFor(method, options) {
     ].join(''),
   };
   if (options['omiseVersion']) {
-    headers['Omise-Version'] = options['omiseVersion'];
+    headers['omiseVersion'] = options['omiseVersion'];
```
Comment (varshard) [inline lib/api.js]: `Omise-Interface` read version header in a `Omise-Version` form. It need to be either `Omise-Version` or `omise-version`. `omiseVersion` won't be readed.

Comment (gelim-ags) [reply lib/api.js]: Changed back to Omise-Version

**Comment by gelim-ags** [comment]
@varshard Comments addressed

**Pull request merged**